### DISCUSSION TITLE
Configuring new local vault from CouchDB

### DISCUSSION THREAD

**Discussion by slimshader** [vrtmrz/obsidian-livesync#420] [Q&A]
Hi, just configured livesync via fly.io and every thing seems to work great (thanks!) but I have a question: I configured secondary valut by generating configuration url from initial vault but how can I create new vault if I loose all the local vaults but still have CouchDB instance running?

**Comment by vrtmrz** [comment]
Thank you for asking me!
The easiest way is to set up with the already configured Vault Setup URI, confirm with `Just import settings`, and then change only the `Database name`. Once CouchDB has been set up, each database will be automatically created on the first connect.
